scan-command/src/main/java/io/ballerina/scan/PlatformPluginContext.java

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ *
+ *  WSO2 LLC. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied. See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package io.ballerina.scan;
+
+import java.util.Map;
+
+/**
+ * {@code PlatformPluginContext} represents a context passed from the scan tool to the platform plugins.
+ *
+ * @since 0.1.0
+ */
+public interface PlatformPluginContext {
+
+    /**
+     * Returns the  platform arguments defined in the Scan.toml file as a map.
```

Comment:
```suggestion
     * Returns the platform arguments defined in the Scan.toml file as a map.
```

Reply:
These are arguments specific to this platform, right? May be better to elaborate.